lib/ai/tools/agent/file-write-tool.ts

```
@@ -27,18 +29,34 @@ const writeFileContent = async (
+      // File doesn't exist yet
+      fileExists = false;
+    }
+
+    if (append && fileExists) {
+      await sandbox.files.write(file, originalContent + finalContent);
+    } else {
+      await sandbox.files.write(file, finalContent);
+    }
+
+    // Determine the operation mode
+    let mode;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)